File: Brute Force/acmicpc.net/2309/Main.java
import java.io.BufferedReader;
import java.io.BufferedWriter;
import java.io.FileInputStream;
import java.io.InputStreamReader;
import java.io.OutputStreamWriter;
import java.util.ArrayList;
import java.util.Collection;
import java.util.Collections;
import java.util.StringTokenizer;

class Main {
    // static final String inputFilePath = "./˝˝˝˝˝˝
    // Structure/Stack/acmicpc.net/1406/sample.txt";
    static final String inputFilePath = "./# format/sample.txt";
    static final int mod = 1000000000;

    public static void main(String[] args) throws Exception {
        System.setIn(new FileInputStream(inputFilePath));
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        ArrayList<Integer> al = new ArrayList<Integer>();
        int sum = 0;
        for (int i = 0; i < 9; i++) {
            al.add(Integer.parseInt(br.readLine()));
            sum += al.get(i);
        }
        int idxI = 0;
        int idxJ = 0;
        outer: for (int i = 0; i < 9; i++) {
            for (int j = 0; j < 9; j++) {
                if(i!=j){
                    int compare = sum;
                    if (compare - al.get(i) - al.get(j) == 100) {
                        idxI = i;
                        idxJ = j;
                        break outer;
                    }
                }
            }
        }
        ArrayList<Integer> answer = new ArrayList<Integer>();        
        for (int i = 0; i < al.size(); i++) {
            if(i!= idxI && i != idxJ)
                answer.add(al.get(i));
        }
        Collections.sort(answer);
        for (int i = 0; i < answer.size(); i++) {
            System.out.println(answer.get(i));
        }
        
    }
}

File: Brute Force/acmicpc.net/2309/Main.js
const inputFilePath = "/sample.txt";
let input = require("fs")
  .readFileSync(__dirname + inputFilePath)
  // .readFileSync('/dev/stdin')
  .toString()
  .trim()
  .split("\n");
let a = input.map(item => {
  return parseInt(item);
})
let sum = 0;
a.forEach(element => sum += element);
let checkArr = [];
outer:
for (let i = 0; i < a.length; i++) {
  for (let j = 0; j < a.length; j++) {
    if (i !== j && sum - a[i] - a[j] === 100) {
      checkArr.push(i);
      checkArr.push(j);
      break outer;
    }
  }
}
let answer = [];
for (let i = 0; i < a.length; i++) {
  if (!checkArr.includes(i)) {
    answer.push(a[i]);
  }
}
answer.sort((a, b) => a - b);
for (let i = 0; i < answer.length; i++) {
  console.log(answer[i]);
}
